swarm/storage/localstore/localstore.go

```diff
@@ -44,7 +44,7 @@ var (
 
 var (
 	// Default value for Capacity DB option.
-	defaultCapacity uint64 = 5000000
+	defaultCapacity uint64 = 30000000
```

Comment:
@jmozah, to reiterate about my comment from the other pull request:
this affects all swarm users, meaning that:
1. every user that runs a swarm node for whatever purpose on their machine (and let us please assume that a lot of those nodes would be unfortunately running on laptops), would have to allocate much more storage for their swarm node. you are now forcing these users to run swarm with another flag to limit their store size otherwise their machines would fill up to a halt.
2. there also also a silent assumption here that all sys-admins that have a swarm node running somewhere (and let us please remember that the various running configurations could be quite different - from kubernetes pods to aws nodes to vagrant/terraform provisioned templates) to adjust their templates and perform a running upgrade with changing all of the underlying partitions/space allocations.
3. by increasing the default store size we also increase the possible data churn on the network for nodes that pop on-and-off the grid (laptops) - these nodes will now contain more data on them, and every time they disconnect from the network, every involved chunk that resides on these machines will render the root hash of the associated content irretrievable and useless. we have touched upon this point in Prague and our plan is to make all nodes default light nodes unless specified otherwise. in my eyes increasing default store size right now is very problematic from this perspective. if anything - we should decrease it
4. as we've already discussed, the problem here is that swarm right now doesn't support high-burst disk space usage - we must alter the localstore to support this, probably by evading the gc index before files are synced, this way the file will still be added to the localstore but the gc won't pick up the chunks for cleanup before they are synced.

5. in any case i strongly resist increasing this.


cc @zelig @nonsense @skylenet @nolash

Reply:
i agree. But it is important to set this for testing this PR in earnest, so I suggest we include a description how to set it via an environment variable or config file

Reply:
Well.. I agree that we should not increase this as part of this PR. I will bring this back to normal levels.

Reply:
Thanks @jmozah <3